src/dcat-us/dataset-formatter.test.ts

```diff
@@ -944,7 +942,7 @@ describe('formatDcatDataset', () => {
     }
   });
 
-  it('overwrites critical fields without values to an empty string', () => {
+  it('does not define spatial property when template does not specify it', () => {
```

Comment:
It looks like you may have deleted the generic `overwrites critical fields` test instead of just adding tests for `spatial`. If that's the case, would you mind adding it back in?

Reply:
It turns out that once you remove `spatial` there are no longer any, at least from what I can tell, non-editable fields that are at the dataset-level, instead they are all feed-level or will never get populated by the dataset (contactPoint[@type]). If this is incorrect i can add back in pretty easily

Reply:
Sorry for the false alarm, you're right. The git diff was a little tricky so I thought that more was in that test.